test/benchmark.rb

```diff
@@ -1,87 +1,28 @@
 # frozen_string_literal: true
 
 require "benchmark/ips"
-require "html/pipeline"
-require "commonmarker"
 require "sanitize"
 require "selma"
-require_relative "benchmark/selma_config"
-
-REWRITE_INPUT = File.read("test/benchmark/rewrite_benchmark_input.md").freeze
-
-def bytes_to_megabytes(bytes)
-  (bytes.to_f / 1_000_000).round(2)
-end
 
 DIR = File.expand_path(File.dirname(__FILE__))
 
 DOCUMENT_HUGE   = File.read("#{DIR}/benchmark/html/document-huge.html").encode("UTF-8", invalid: :replace, undef: :replace)
 DOCUMENT_MEDIUM = File.read("#{DIR}/benchmark/html/document-medium.html").encode("UTF-8", invalid: :replace, undef: :replace)
 DOCUMENT_SMALL  = File.read("#{DIR}/benchmark/html/document-small.html").encode("UTF-8", invalid: :replace, undef: :replace)
 
-FRAGMENT_LARGE = File.read("#{DIR}/benchmark/html/fragment-large.html").encode("UTF-8", invalid: :replace, undef: :replace)
-FRAGMENT_SMALL = File.read("#{DIR}/benchmark/html/fragment-small.html").encode("UTF-8", invalid: :replace, undef: :replace)
-
 def compare_sanitize
   sanitize_config = Sanitize::Config::RELAXED
-  [[DOCUMENT_HUGE, "huge"], [DOCUMENT_MEDIUM, "medium"], [DOCUMENT_SMALL, "small"]].each do |(html, label)|
     Benchmark.ips do |x|
-      x.report("sanitize-document-#{label}") do
+      x.report("sanitize-#{label}") do
         Sanitize.document(html, sanitize_config)
       end
 
-      x.report("selma-document-#{label}") do
-        Selma::HTML.new(html, sanitize: Selma::Sanitizer::Config::RELAXED).rewrite
+      x.report("selma-#{label}") do
+        sanitizer = Selma::Sanitizer.new(Selma::Sanitizer::Config::RELAXED)
+        Selma::Rewriter.new(sanitizer: sanitizer).rewrite(html)
       end
     end
   end
 end
 
-def compare_rewriting
```

Comment:
I removed the rewriting benchmark because I can't find the Selma handlers that it's calling.